_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -70,9 +76,28 @@ On the **target 3DS** (the 3DS that you are trying to modify):
 1. Select "Start" on the **source 3DS** once it has detected the **target 3DS**
 1. Once multiplayer has loaded, navigate to `Grand Prix` -> `50cc` -> (any driver) -> `Mushroom Cup` -> `OK`
 1. Wait a while (a percentage should be displayed on the **source 3DS**)
-1. If the exploit was successful, the **target 3DS** will have booted into SafeB9SInstaller
+1. If the exploit was successful, the **target 3DS** will have booted into the 3DS ROP xPloit Injector
+1. Press (X) to use unSAFE_MODE
+  + The target 3DS will automatically shut down
+  + You won't need the source 3DS to complete any further steps on this guide
 
-#### Section IV - Installing boot9strap
+#### Section IV - unSAFE_MODE
+
+If your (Right/Left Shoulder), (D-Pad Up), or (A) buttons do not work, you will not be able to follow the instructions. For assistance with this matter, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask, in English, for help.
+{: .notice--warning}
+
+1. With your device still powered off, hold the following buttons: (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A), then press (Power)
```

Comment:
This step needs to be updated to align with the recently merged pull request on HBL-USM.

See the other comment's suggestion on just copy-pasting everything.